plotter/boxplot_test.go

```diff
@@ -0,0 +1,213 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+)
+
+func ExampleBoxPlot() {
+	// Create the sample data.
+	rand.Seed(int64(0))
```

Comment:
rand.Seed(0)

Note that the standard rand seed is 1 (initially package rand seeds with 1).